test_apps/smokeTests/gridTools.cpp

```diff
@@ -334,6 +360,23 @@ bool RunTest(Grid *grid, bool silenceTime)
 
     PrintStats(rms, numMissingValues, disagreements, time, silenceTime);
 
+    // Test OpenMP implementation of Grid::GetRange()
```

Comment:
Are missing values being tested? Given the logic in the code for missing value handling it seems pretty important that data with missing values gets tested.

Reply:
I tested using real data (MOM and ROM) you suggested earlier. Any missing values in these artificial grids are also tested.

Reply:
@sgpearse do we have missing values in the test grids? It would be a good idea to include if we don't. Perhaps spray some randomly into each test grid? Having a grid entirely of missing values would also be good test.

Reply:
@clyne we currently do not, so I've added them in PR #3096.  The tests include randomly assigning certain values as missing, and testing each grid type when they contain all missing values.  This seems to have turned up some bugs in the Stretched, Layered, and Curvilinear grid tests regarding RMS error.

Reply:
> @clyne we currently do not, so I've added them in PR #3096. The tests include randomly assigning certain values as missing, and testing each grid type when they contain all missing values. This seems to have turned up some bugs in the Stretched, Layered, and Curvilinear grid tests regarding RMS error.

Indeed it has. I'll add a comment to #3096